setup.py

```diff
@@ -3,4 +3,4 @@
 import setuptools
 
 setuptools.setup(use_scm_version=True,
-                 packages=setuptools.find_packages(exclude=['examples', 'tests']), )
+                 packages=setuptools.find_packages(exclude=['tests']), )
```

Comment:
```suggestion
                 packages=setuptools.find_packages(exclude=['tests*']), )
```

There needs to be also the star to exclude sub packages like `tests.unit`.

Reply:
neat, I didn't know it accepts wildcards

Reply:
I've spent some time with that for ogr...